examples/boxplot.rs

```diff
@@ -0,0 +1,222 @@
+        .iter()
+        .map(|x| x.2.values().to_vec())
+        .flatten()
+        .collect();
+    let values_range = fitting_range(values.iter());
+
+    let mut chart = ChartBuilder::on(&upper)
+        .x_label_area_size(40)
+        .y_label_area_size(120)
+        .caption("Ping Boxplot", ("sans-serif", 20).into_font())
+        .build_ranged(
+            values_range.start - 1.0..values_range.end + 1.0,
+            category.range(),
+        )?;
+
+    chart
+        .configure_mesh()
+        .x_desc("Ping, ms")
+        .y_desc(category.name())
+        .y_labels(category.len())
+        .line_style_2(&WHITE)
+        .draw()?;
+
+    for (label, (values, style, offset)) in &series {
+        chart
+            .draw_series(values.iter().map(|x| {
+                Boxplot::new_horizontal(category.get(&x.0).unwrap(), &x.1)
+                    .width(20)
+                    .whisker_width(0.5)
+                    .style(style)
+                    .offset(*offset)
+            }))?
+            .label(label)
+            .legend(move |(x, y)| Rectangle::new([(x - 5, y - 5), (x + 5, y + 5)], style.filled()));
+    }
+    chart
+        .configure_series_labels()
+        .position(SeriesLabelPosition::UpperRight)
+        .background_style(WHITE.filled())
+        .border_style(&BLACK.mix(0.5))
+        .draw()?;
+
+    let drawing_areas = lower.split_evenly((1, 2));
+    let (left, right) = (&drawing_areas[0], &drawing_areas[1]);
+
+    let quartiles_a = Quartiles::new(&[
+        6.0, 7.0, 15.9, 36.9, 39.0, 40.0, 41.0, 42.0, 43.0, 47.0, 49.0,
+    ]);
+    let quartiles_b = Quartiles::new(&[16.0, 17.0, 50.0, 60.0, 40.2, 41.3, 42.7, 43.3, 47.0]);
+    let category_ab = Category::new("", vec!["a", "b"]);
+    let values_range = fitting_range(
+        quartiles_a
+            .values()
+            .iter()
+            .chain(quartiles_b.values().iter()),
+    );
+    let mut chart = ChartBuilder::on(&left)
+        .x_label_area_size(40)
+        .y_label_area_size(40)
+        .caption("Vertical Boxplot", ("sans-serif", 20).into_font())
+        .build_ranged(
+            category_ab.get(&"a").unwrap()..category_ab.get(&"b").unwrap(),
+            values_range.start - 10.0..values_range.end + 10.0,
+        )?;
+
+    chart.configure_mesh().line_style_2(&WHITE).draw()?;
+    let area = chart.plotting_area();
+    area.draw(&Boxplot::new_vertical(
+        category_ab.get(&"a").unwrap(),
+        &quartiles_a,
+    ))?;
+    area.draw(&Boxplot::new_vertical(
+        category_ab.get(&"b").unwrap(),
+        &quartiles_b,
+    ))?;
+
+    let mut chart = ChartBuilder::on(&right)
+        .x_label_area_size(40)
+        .y_label_area_size(40)
+        .caption("Horizontal Boxplot", ("sans-serif", 20).into_font())
+        .build_ranged(-30f32..90f32, 0..3)?;
+
+    chart.configure_mesh().line_style_2(&WHITE).draw()?;
+    let area = chart.plotting_area();
+    area.draw(&Boxplot::new_horizontal(1, &quartiles_a))?;
```

Comment:
Just wondering if we can use `draw_series` here, that won't be a very big difference

Reply:
Sure, I'll change it, but wondering why do you prefer `draw_series` over `draw` here (as we're not drawing series here, but rather just one element)?